Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: generate build
+
+.DEFAULT_GOAL := generate
+
+VERSION := $(shell git tag | sort -V | tail -n 1 | cut -c2-)
+
+ifeq ($(shell uname -s),Darwin)
+    ifeq ($(shell uname -m),arm64)
+        ARCHITECTURE := :osx-x86_64e
```

Comment:
Why do we want to distiguish build environment according hosting machine type? I do not see point. This is probably reason, why java cannot be build on MacOS, because you would have to change gradle plugin. Lets discuss it f2f.